From: Raul Rodrigo <raurodse@gmail.com>
Date: Wed, 12 May 2021 16:24:59 +0200
Subject: apply patch

---
 src/modules/networkcfg/main.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/modules/networkcfg/main.py b/src/modules/networkcfg/main.py
index 608cfd2..40d1530 100644
--- a/src/modules/networkcfg/main.py
+++ b/src/modules/networkcfg/main.py
@@ -59,7 +59,7 @@ def run():
                 continue
 
             try:
-                shutil.copy(source_network, target_network)
+                shutil.copy(source_network, target_network, follow_symlinks=False)
             except FileNotFoundError:
                 libcalamares.utils.debug(
                     "Can't copy network configuration files in "
@@ -80,7 +80,7 @@ def run():
                 )
 
         try:
-            shutil.copy(source_resolv, target_resolv)
+            shutil.copy(source_resolv, target_resolv, follow_symlinks=False)
         except Exception as err:
             libcalamares.utils.debug(
                 "Can't copy resolv.conf from {}: {}".format(source_resolv, err)
