From: Raul Rodrigo Segura <raurodse@gmail.com>
Date: Wed, 30 Jun 2021 22:05:21 +0200
Subject: add patch to install packages on live

---
 src/modules/networkcfg/main.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/modules/networkcfg/main.py b/src/modules/networkcfg/main.py
index 40d1530..bf0a450 100644
--- a/src/modules/networkcfg/main.py
+++ b/src/modules/networkcfg/main.py
@@ -18,6 +18,8 @@ import shutil
 import libcalamares
 
 import gettext
+from pathlib import Path
+
 _ = gettext.translation("calamares-python",
                         localedir=libcalamares.utils.gettext_path(),
                         languages=libcalamares.utils.gettext_languages(),
@@ -81,6 +83,12 @@ def run():
 
         try:
             shutil.copy(source_resolv, target_resolv, follow_symlinks=False)
+            x = Path(source_resolv)
+            if x.is_symlink():
+                y = Path(root_mount_point)
+                dest = y.joinpath(x.resolve().relative_to("/"))
+                dest.parent.mkdir(parents=True,exist_ok=True)
+                shutil.copy(str(x.resolve()),str(dest))
         except Exception as err:
             libcalamares.utils.debug(
                 "Can't copy resolv.conf from {}: {}".format(source_resolv, err)
