From: Raul Rodrigo <raurodse@gmail.com>
Date: Mon, 5 Jul 2021 15:32:32 +0200
Subject: replace pkexec with sudo

---
 calamares.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/calamares.desktop b/calamares.desktop
index 53f32bd..ed2652e 100644
--- a/calamares.desktop
+++ b/calamares.desktop
@@ -5,7 +5,7 @@ Name=Install System
 GenericName=System Installer
 Keywords=calamares;system;installer;
 TryExec=calamares
-Exec=sh -c "pkexec calamares"
+Exec=sh -c "sudo calamares"
 Comment=Calamares — System Installer
 Icon=calamares
 Terminal=false
